api/systems/cable/cable-ctl.ts

```diff
@@ -28,7 +27,7 @@ function CableCTL(
     PrimaryProduct + ResidueRecoveredOptional;
 
   // Limits
-  const InLimits1 = InLimits(input, intermediate);
+  const InLimits1 = 1;
```

Comment:
i know it'll be a little annoying, but it's better to remove references to `InLimits` if we're not going to use it. i would delete it from all of these files